### PR TITLE
CONSOLE-3334: Update copiedCSVsDisabled to contain managed clusters

### PR DESCRIPTION
**Pull request by rhamilto**
Includes changes from https://github.com/openshift/console/pull/12372

### CODE REVIEW

File: cmd/bridge/main.go
Comment (rhamilto) [inline cmd/bridge/main.go:64]: See https://github.com/openshift/console/pull/12372#discussion_r1049798797

**Comment by rhamilto** [comment]
I tested this against a standalone cluster and a multi-cluster console.  Everything seems to be working as expected.

**Comment by rhamilto** [comment]
/hold

unit tests are broken

**Comment by rhamilto** [comment]
/hold cancel

I believe I fixed the failing unit tests.

**Comment by rhamilto** [comment]
QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

**Comment by rhamilto** [comment]
Console approver:
/assign @TheRealJon

**Review by TheRealJon — APPROVED** [review]
Looks good!  The only nit I have is that we could make this a little more "DRY" if we create a `useCopiedCSVsDisabled` hook.

/lgtm

**Review by opayne1 — COMMENTED** [review]
/label docs-approved

**Comment by rhamilto** [comment]
@yapei, can you please review?

**Comment by rhamilto** [comment]
@yapei, can you try testing these changes independently?  I think https://issues.redhat.com/browse/OCPBUGS-5901 was caused by unrelated changes in the testathon environment.

**Comment by rhamilto** [comment]
https://issues.redhat.com/browse/OCPBUGS-5489 is blocking @yapei from testing.

**Comment by yapei** [comment]
some questions and issues 
1. it looks like `window.SERVER_FLAGS.copiedCSVsDisabled` value is not syncing with cm/managed-clusters, for example in `cm/managed-clusters` we only have one managed cluster `jfrancoa-175074` with `copiedCSVsDisabled: false`,  however 
`window.SERVER_FLAGS.copiedCSVsDisabled` still have value
```
{ "groupd-174971": false, "jfrancoa-175074": false, "local-cluster": false }
```
(groupd-174971 was imported and detached, we removed the entry from cm/managed-clusters but it still appears in 'All Clusters' dropdown
2.

**Comment by yapei** [comment]
one question about how `window.SERVER_FLAGS.copiedCSVsDisabled` is synced and updated, it looks like it is not consistent with `cm/managed-clusters` 
Screenshot 2023-02-02 at 6 02 08 PM

**Comment by rhamilto** [comment]
@TheRealJon, @jhadvig, please see @yapei's comment above.

**Comment by yapei** [comment]
https://issues.redhat.com/browse/OCPBUGS-5901 still can be reproduced when PR changes are included

**Comment by rhamilto** [comment]
QE verification blocked by https://issues.redhat.com/browse/OCPBUGS-5901

**Comment by yapei** [comment]
since OCPBUGS-5901 is unrelated to this PR change(other changes look good), I will approve the PR to get it merged
/label qe-approved

**Pull request merged**
